Code/GraphMol/DistGeomHelpers/Wrap/testDistGeom.py

```diff
@@ -375,16 +375,24 @@ def test8MultiThreadMultiConf(self):
             ee = ff.CalcEnergy()
             energies.append(ee)
 
-        mol = Chem.AddHs(Chem.MolFromSmiles("CC(C)(C)c(cc12)n[n]2C(=O)/C=C(N1)/COC"))
-        cids = rdDistGeom.EmbedMultipleConfs(mol,200,maxAttempts=30,randomSeed=100,numThreads=4)
+        mol2 = Chem.AddHs(Chem.MolFromSmiles("CC(C)(C)c(cc12)n[n]2C(=O)/C=C(N1)/COC"))
+        cids2 = rdDistGeom.EmbedMultipleConfs(mol2,200,maxAttempts=30,randomSeed=100,numThreads=4)
+        self.assertTrue(lstEq(cids, cids2))
         nenergies = []
-        for cid in cids:
-            ff = ChemicalForceFields.UFFGetMoleculeForceField(mol, 10.0, cid)
+        for cid in cids2:
+            ff = ChemicalForceFields.UFFGetMoleculeForceField(mol2, 10.0, cid)
             ee = ff.CalcEnergy()
             nenergies.append(ee)
 
-        self.assertTrue(lstEq(energies, nenergies,tol=1e-6))
```

Comment:
Double-checked it now. The difference is 0.44 kcal/mol. These are the respective coordinates:

```
   -5.1797    2.2474   -0.6697 C   -5.1811    2.2468   -0.6685 C
   -5.6290    1.2275    0.3451 C   -5.6310    1.2260    0.3444 C
   -5.5224    1.8514    1.7061 C   -5.5220    1.8483    1.7065 C
   -7.0562    0.8384   -0.0095 C   -7.0584    0.8397   -0.0088 C
   -4.8006   -0.0136    0.3089 C   -4.8017   -0.0156    0.3087 C
   -4.5736   -0.6920   -0.8832 C   -4.5730   -0.6915   -0.8842 C
   -3.3172   -1.2121   -1.1726 C   -3.3175   -1.2140   -1.1726 C
   -3.9426   -0.3588    1.3469 C   -3.9441   -0.3591    1.3471 C
   -2.7622   -1.0483    1.0988 C   -2.7624   -1.0460    1.0983 C
   -2.3242   -1.2585   -0.2018 C   -2.3238   -1.2571   -0.2025 C
   -0.9182   -1.0776   -0.5858 C   -0.9172   -1.0750   -0.5837 C
   -0.3951   -0.0381   -1.3450 C   -0.3956   -0.0361   -1.3449 C
    0.8604    0.2780   -0.8402 C    0.8598    0.2786   -0.8397 C
    0.1528   -1.5939    0.0592 N    0.1537   -1.5930    0.0591 N
    1.2284   -0.8429   -0.1726 N    1.2289   -0.8419   -0.1722 N
    2.5272   -0.9422    0.2013 C    2.5272   -0.9418    0.2021 C
    2.9708   -1.9864    0.7232 O    2.9706   -1.9851    0.7241 O
    3.3658    0.1615    0.1997 C    3.3673    0.1619    0.1989 C
    2.8315    1.3626   -0.2328 C    2.8331    1.3627   -0.2352 C
    1.6775    1.3538   -0.9412 N    1.6788    1.3536   -0.9429 N
    4.8112   -0.0909    0.1674 C    4.8121   -0.0913    0.1668 C
    5.2796   -1.2057   -0.2064 O    5.2801   -1.2064   -0.2050 O
    5.6878    0.9920    0.1003 O    5.6880    0.9913    0.0982 O
    6.9883    0.8754   -0.4714 C    6.9890    0.8734   -0.4703 C
    8.0398    1.1732    0.5699 C    8.0391    1.1715    0.5709 C
```

So differences are indeed quite small. But `tol=5.0` is probably overkill - I just modified the PR to `tol=1.0`.